Dockerfile

```
@@ -1,24 +1,28 @@
-FROM golang AS builder
-
-COPY . /go/src/github.com/rclone/rclone/
-WORKDIR /go/src/github.com/rclone/rclone/
-
-RUN \
-  CGO_ENABLED=0 \
-  make
-RUN ./rclone version
-
-# Begin final image
-FROM alpine:latest
-
+ARG ALPINE_VERSION=3.18
+ARG GO_VERSION=1.19
```

Changing this to 1.21 (as it is on the master branch), to keep from having any funny surprise. It will however need to be bumped once we start using 1.22 language features or bump go.mod to 1.22 at a later time.

```suggestion
ARG GO_VERSION=1.21
```